tests/simpleKVBC/cmf/skvbc_messages.cmf

```diff
@@ -0,0 +1,86 @@
+# Definition of client-server interface for the SimpleKVBC test application.
+#
+# Note SimpleKVBC effectively implements a versioned key-value store supporting
+# only keys and values that are byte strings of exactly 21 bytes.
+#
+# Note that, when key-value pairs (of 21-byte byte strings) need to be handled
```

Comment:
I understand the goal here is to maintain backwards compatibility, but I'm not sure it's useful in this case. I believe the 21 byte form only existed because it was easier to pick a static allocation size for whoever wrote the code initially.  I think it would be beneficial to the system as a whole if instead the key value pairs could be arbitrary sizes (with some max limits). I don't think there's many specific cases where we rely on 21 byte keys and values for any reason other than "sizing". Since we get that sizing for free with CMF, we can probably do away with the limitation altogether.

Reply:
I have decided to adopt this suggestion in my most recent revisions to this PR; the bytestrings in the CMF messages this PR adds for SKVBC should now be arbitrary-length and represented with the CMF primitive `bytes` type. I have considered this a bit, and I think this may increase the scope of my upcoming changes to actually switch SKVBC and Apollo to actually use the CMF messages I am defining in this PR, as more of the existing SKVBC implementation will need to be refactored to accommodate arbitrary-length byte strings, but I guess now is a good a time as any to make this revision and I think the SKVBC code will be cleaner and more modern for it.